src/main/scala/chargepoint/docile/dsl/OcppTest.scala

```diff
@@ -29,9 +30,15 @@ trait OcppTest extends MessageLogging {
     chargerId: String,
     endpoint: URI,
     version: Version,
-    authKey: Option[String]
+    authKey: Option[String],
+    keystoreFile: Option[String],
+    keystorePassword: Option[String]
   ): Unit = {
-    connect(receivedMsgManager, chargerId, endpoint, version, authKey)
+    connect(receivedMsgManager, chargerId, endpoint, version, authKey)(
```

Comment:
OcppTest is kind of becoming a God object now that it also has to know about TLS...

I don't immediately have a full solution for that, but at least we an alleviate the pain by turning these two keystore-related parameters into one SSLContext in Main.scala already, and then just pass in an SSLContext in to OcppTest.